client/components/sidebar.vue

```diff
@@ -119,23 +221,23 @@
           if (!el) {
             return null;
           }
-          return Object.assign({}, c, {highlightTop: getOffset(el).top});
+          return Object.assign({}, c, {
+            highlightTop: getOffset(el).top,
+            showDetails: false,
+            showAddCommentForm: false,
+            showAllReplies: c.replies.length <= 1,
+            hasManyReplies: c.replies.length > 1,
+            isReply: c.replyTo === null,
+            focus: false,
+          });
         }).filter((c) => {
           return c;
         }).sort((a, b) => {
           // We sort these values in order to place the comments correctly.
           // For example, you could have a newer comment positioned on top
           // of an older one, so we have to sort the positions accordinglys
           // before Vue tries to render them.
-          if (a.highlightTop > b.highlightTop) {
-            return 1;
-          }
-          else if (a.highlightTop < b.highlightTop) {
-            return -1;
-          }
-          else {
-            return 0;
-          }
+          return a.highlightTop - b.highlightTop;
```

Comment:
So this is not sorted anymore by created time?

Reply:
Fixed. The comments are sorted by createdAt field again.

Reply:
So, I am not sure by what they should be sorted. :-) Probably it does not matter just that it is deterministic, no?